cupy/random/generator.py

```diff
@@ -325,6 +325,16 @@ def seed(self, seed=None):
 
         self.rk_seed = numpy.uint64(seed)
 
+    def standard_exponential(self, size=None, dtype=float):
+        """Returns an array of samples drawn from the standard exp distribution.
+
+         .. seealso::
+            :func:`cupy.random.standard_exponential` for full documentation,
+            :meth:`numpy.random.RandomState.standard_exponential`
+        """
+        x = self._random_sample_raw(size, dtype)
+        return cupy.log(x)
```

Comment:
`return cupy.log(x, out=x)` might improve the performance.  Could you check it?

Reply:
A little improvement was seen.

before
```
>>> timeit.timeit('cupy.random.standard_exponential(size=10**7)', globals=globals(), number=1000)
0.30956048937514424
```

after
```
>>> timeit.timeit('cupy.random.standard_exponential(size=10**7)', globals=globals(), number=1000)
0.309205976780504
```